rcon/rcon.py

```diff
@@ -1312,6 +1314,99 @@ def set_maprotation(self, map_names: list[str]) -> list[Layer]:
             super().remove_map_from_rotation(current[0], 1)
 
         return self.get_map_rotation()
+    
+    @ttl_cache(ttl=10)
+    def get_objective_row(self, row: int):
+        return super().get_objective_row(row)
+    
+    def get_objective_rows(self) -> List[List[str]]:
+        try:
+            return [
+                self.get_objective_row(row)
+                for row in range(5)
+            ]
+        except CommandFailedError:
```

Comment:
I'm not quite sure if we should default to an empty list here. A proper error message would probably make more sense. Especially as there might be errors, which are not related to the map not having objective rows (like Skirmish). What do you think?